core/src/tui_core/styles/stylesheet.rs

```diff
@@ -73,3 +73,63 @@ impl Stylesheet {
     }
   }
 }
+
+/// Macro to make building `StyleSheet` easy
+///
+/// Use case:
+/// ```rust
```

Comment:
This doc test doesn't run. So just replace:
```text
```rust
```
with 

```text
```ignore
```

Doc tests in our codebase are difficult to get running since there are so many dependencies that need to be brought along. We handle the actual tests in the `tests` folder, so these docs end up being snippets that the developer using the library can easily copy / paste, or get some ideas on how to use this function.